generic/restapi/v0/README.mdx

```diff
@@ -46,11 +46,54 @@ ${connection.<my-connection-id>}`.
 
 | Field | Field ID | Type | Note |
 | :--- | :--- | :--- | :--- |
-| Authentication (required) | `authentication` | object | Authentication method to use for the REST API |
+| Authentication (required) | `authentication` | object | Authentication method to use for the REST API  |
 
 
 
 
+
+
+<details>
+<summary> Authentication </summary>
+
+#### Authentication
+
+`authentication` must fulfill one of the following schemas:
+
+##### NO_AUTH
```

Comment:
Like this?
<img width="597" alt="image" src="https://github.com/user-attachments/assets/7c249384-1766-4887-abaf-94c0701b831c">

Reply:
So adding the code styling is better, if you want to merge this PR it won't be a blocker. My suggestion in

> I'd add a title to each of the properties

Was adding a title to each `oneOf` property and using it instead of the `const` value. It turns out that in the Rest API case the title is there (though it could be improved to be `No Authentication` / `Basic Authentication` / etc.:

```json
          {
            "properties": {
              "auth-type": {
                "const": "NO_AUTH",
                "description": "No Authentication",
                "instillUIOrder": 0,
                "order": 0,
                "type": "string"
              }
            },
            "required": [
              "auth-type"
            ],
            "title": "No Auth"
          }
```

Reply:
<img width="1321" alt="image" src="https://github.com/user-attachments/assets/e2dfef0c-4a44-4bcc-a4c6-2e11d41b5209">
Thanks for advising! Now, it will be like this!